python/raft-dask/setup.py

```diff
@@ -26,10 +26,10 @@
     "numpy",
     "numba>=0.49",
     "joblib>=0.11",
-    "dask-cuda>=22.10",
-    "dask>=2022.9.1",
+    "dask-cuda==22.12",
```

Comment:
We might want to add this to release version upgrade script at https://github.com/rapidsai/raft/blob/bd2fa8ade237613c769aaf26489a7bf49b353d8c/ci/release/update-version.sh

Though this can be done in 23.02, should be good to merge this PR once @cjnolet approves.